### PR TITLE
[Bugfix] Update local tile aspect ratio and fix CallWithChat flashing with scrollable gallery

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Updates aspect ratio of local video tile on mobile to show more of local feed

add CallCompositeContainer width
# Why

Stops the CallComposite from causing flashing in the CWC composite when on mobile 
Broken:
image
Fixed:
image

Composite window would stretch to show all of the scrollable gallery tiles in CWC composite causing flashing.

https://skype.visualstudio.com/SPOOL/_workitems/edit/3138347
# How Tested

Tested locally on chrome and on local phone device to verify flashing in CWC composite is gone.

### CODE REVIEW

**Comment by JamesBurnside** [comment]
Something looks a tad wrong here:
image

**Comment by dmceachernmsft** [comment]
> Something looks a tad wrong here: image

is a screen shot of when it was broken to show the growing. does not look like that now 😅

**Comment by edwardlee-msft** [comment]
> Something looks a tad wrong here: image

I think this is the issue. Not a picture of the fix. 
@dmceachernmsft could you upload a picture with the fix implemented as well?

File: packages/react-components/src/components/VideoGallery/styles/VideoGalleryResponsiveHorizontalGallery.styles.ts [hunk 1]
```diff
@@ -36,7 +36,8 @@ export const horizontalGalleryStyle = (isNarrow: boolean): HorizontalGalleryStyl
  * Small horizontal gallery tile size in rem
  * @private
  */
-export const SMALL_HORIZONTAL_GALLERY_TILE_SIZE_REM = { height: 5.5, width: 5.5 };
+export const SMALL_HORIZONTAL_GALLERY_TILE_SIZE_REM = { height: 6.5, width: 6.5 };
```
Comment (edwardlee-msft) [inline packages/react-components/src/components/VideoGallery/styles/VideoGalleryResponsiveHorizontalGallery.styles.ts:39]: is this an intended change

**Comment by dmceachernmsft** [comment]
Uploaded a photo of it fixed as well in the description, sorry I forgot to put that in there as well! 😊

**Review by edwardlee-msft — APPROVED** [review]

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**